controllers/barbicanapi_controller.go

```diff
@@ -289,6 +299,21 @@ func (r *BarbicanAPIReconciler) generateServiceConfigs(
 		"EnableSecureRBAC": instance.Spec.EnableSecureRBAC,
 	}
 
+	// create httpd  vhost template parameters
+	httpdVhostConfig := map[string]interface{}{}
+	for _, endpt := range []service.Endpoint{service.EndpointInternal, service.EndpointPublic} {
+		endptConfig := map[string]interface{}{}
+		endptConfig["ServerName"] = fmt.Sprintf("barbican-%s.%s.svc", endpt.String(), instance.Namespace)
```

Comment:
I'm not sure what you mean here, I've looked at the other operators and they seem to follow this pattern.  I have noticed that the barbican operator is double named, and maybe this will resolve that.

Reply:
there is a const `barbican.ServiceName` which is used when the service gets created. best to use the same. I know that I have not done this in all of my PRs. we need to follow up on this where a const is used for the service.

https://github.com/openstack-k8s-operators/barbican-operator/blob/main/pkg/barbican/const.go#L7

Reply:
I see, so something like:
`endptConfig["ServerName"] = fmt.Sprintf("%s-%s.%s.svc", barbican.ServiceName, endpt.String(), instance.Namespace)`

Reply:
yes , correct